test/JDBC/upgrade/latest/schedule

```diff
@@ -407,5 +407,6 @@ BABEL-3657
 test_windows_alter_user
 BABEL-733
 BABEL-3938
+BABEL-3478
```

Comment:
this only for Major version upgrade without any minor (3.1 to 3.1). If there is no schedule for 3 to 3.1, add it one of the MVU schedule like 14.7 to 3.1.